spec.html

```diff
@@ -92,38 +92,54 @@
     UnicodePropertyValueCharacters</ins>
 </emu-grammar>
 
+<emu-clause id="sec-static-semantics-sourcetext">
+  <h1>Static Semantics: SourceText</h1>
+  <emu-grammar>UnicodePropertyName :: UnicodePropertyNameCharacters</emu-grammar>
+  <emu-alg>
+    1. Return the source text that was recognized as |UnicodePropertyName|.
```

Comment:
I'd say:

Return the ordered List of the Unicode code points of the source text...

We're moving from the domain of the grammar to the domain of pseudo-code where  we manipulate Lists.